scripts/tiling/stitching.py

```diff
@@ -0,0 +1,256 @@
+""" Script for creating a stitched ROI MIBItiff from a folder of FOVs
+"""
+
+import numpy as np
+import os
+from mibidata import mibi_image as mi
+from mibitracker.request_helpers import MibiRequests
+from collections import OrderedDict
+import time
+from mibidata import tiff
+import json
+
+MAX_TRIES = 10
+
+
+def combine_entity_by_name(roi_fov_paths, cols, rows, enforce_square):
+    min_col = np.min(cols)
+    min_row = np.min(rows)
+    cols=[v-min_col+1 for v in cols]
+    rows=[v-min_row+1 for v in rows]
+    w = np.max(cols)
+    h = np.max(rows)
+
+    panel = None
+    out_img = None
+    for fov_i,roi_fov_path in enumerate(roi_fov_paths):
+        print(fov_i+1,roi_fov_path)
+
+        fov_img = tiff.read(roi_fov_path)
+        panel = fov_img.channels
+        fov_img = fov_img.data
+
+        img_shape = list(np.shape(fov_img))
+        ch_count = np.min(img_shape)
+        shape_2d = img_shape[:-1]
+
+        if out_img is None:
+            out_img = np.zeros((h*shape_2d[0], w*shape_2d[1], ch_count), dtype=fov_img.dtype)
+            out_img_shape = list(np.shape(out_img))
+
+        for ch_i in range(ch_count):
+            out_img[((rows[fov_i]-1)*shape_2d[0]):(rows[fov_i]*shape_2d[0]), 
+                    ((cols[fov_i]-1)*shape_2d[1]):(cols[fov_i]*shape_2d[1]), ch_i] = fov_img[:,:,ch_i]
+    
+    if enforce_square:
```

Comment:
Is `enforce_square` needed after @therealjacinto merges in changes that enable non-square TIFF writing?